packages/sit-onyx/src/components/OnyxDataGrid/features/sorting/SortAction.vue

```diff
@@ -0,0 +1,41 @@
+<script setup lang="ts">
+import arrowSmallDown from "@sit-onyx/icons/arrow-small-down.svg?raw";
+import arrowSmallUp from "@sit-onyx/icons/arrow-small-up.svg?raw";
+import arrowsSort from "@sit-onyx/icons/arrows-sort.svg?raw";
+import { computed } from "vue";
+import { OnyxIconButton } from "../../../..";
+import { injectI18n } from "../../../../i18n";
+import type { SortDirection } from "./sorting";
+import { nextSortDirection } from "./sorting";
+
+const props = defineProps<{
+  /**
+   * Label of this Column.
+   */
+  columnLabel: string;
+  /**
+   * The current sorting direction, that should be indicated.
+   */
+  sortDirection?: SortDirection;
+}>();
+
+const { t } = injectI18n();
+
+const icon = computed(() =>
+  props.sortDirection === "asc"
+    ? arrowSmallUp
+    : props.sortDirection === "desc"
+      ? arrowSmallDown
+      : arrowsSort,
+);
+
+const buttonLabel = computed(() => {
+  const nextDirection = nextSortDirection(props.sortDirection);
+  return t.value(`data-grid.head.sorting-action-${nextDirection}`, {
+    field: props.columnLabel,
+  });
+});
+</script>
+<template>
+  <OnyxIconButton color="neutral" :icon="icon" size="12px" :label="buttonLabel" />
```

Comment:
```suggestion
  <OnyxIconButton color="neutral" :icon="icon" :label="buttonLabel" />
```

The icon button does not have a size property (yet).

Also, are you sure that this is the correct design? Looks quiet big + no gap on hover
![image](https://github.com/user-attachments/assets/9fde4cbd-6104-4570-b357-e1bb46703bff)

![image](https://github.com/user-attachments/assets/2b30856f-b52d-4855-a829-df8410761ee9)